recipes/MEAutility/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MEAutility" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cd1f723992aa7366f1cbbf8f2843e1f61d8123b459435653860a3b544bc752f"
+
+build:
```

Comment:
It looks like this could be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python